marketplace/netlify-build/extension.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "netlify-button",
+  "name": "Netlify Build and Preview",
+  "src": "http://localhost:1234",
+  "sidebar": true,
+  "parameters": {
+    "installation": [
+      {
+        "id": "netlifySiteIds",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify Site Id list (max 3)",
+        "description": "Example: 14234e3,143248,35233",
+        "required": true
+      },
+      {
+        "id": "buildHookUrls",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify build hook URLs (max 3)",
+        "description": "Example: https://api.netlify.com/build_hooks/xyz,https://api.netlify.com/build_hooks/abc",
```

Comment:
Replace xyz, abc with values that resemble Netlify Build hook URLs

Reply:
Done.